Assignment3/src/MenuActions.java

```diff
@@ -0,0 +1,178 @@
+import javax.swing.*;
+import java.awt.*;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class MenuActions {
+
+    public JTextPane textPane = new JTextPane();
+    private ArrayList<String> recentFiles = new ArrayList<>();
+    
+    //constructor
+    public MenuActions(){ }
+
+    //returns textpane object
+    public JTextPane getjTextPane(){
```

Comment:
Good way to pass JPane elements between classes-  I couldn't think of how to do this properly